server/game/cards/02.1-NTNR/ParalysisMark.js

```diff
@@ -0,0 +1,28 @@
+const GameActions = require('../../GameActions/index.js');
+const SpellCard = require('../../spellcard.js');
+
+class ParalysisMark extends SpellCard {
+    setupCardAbilities(ability) {
+        this.spellAction({
+            title: 'Paralysis Mark',
+            playType: 'noon',
+            cost: ability.costs.bootSelf(),
+            target: {
+                activePromptTitle: 'Select a dude to paralyze',
+                waitingPromptTitle: 'Waiting for opponent to select a dude',
+                cardCondition: { condition: card => card.gamelocation === this.gamelocation },
+                cardType: ['dude']
```

Comment:
I am working on cannot effects and it seems we will have to update all the card abilities targeting that do specific actions (e.g. ace, boot, discard and so on). There is a parameter `gameAction` missing that is required for game engine to know the purpose for targeting so it can prevent targeting cards that cannot be aced, booted and so on. From now on, we have to add this property. For this card, it would be:
```javascript
target: {
    activePromptTitle: 'Select a dude to paralyze',
    waitingPromptTitle: 'Waiting for opponent to select a dude',
    cardCondition: { condition: card => card.gamelocation === this.gamelocation },
    cardType: ['dude'],
    gameAction: 'boot'
}
```